### PR TITLE
Added migration info from notifications to alerts in the SDK

### PR DESCRIPTION
**Pull request by kebringer-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [X] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [ ] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [X] A self-review of the code has been done.

## Pull Request Description

Adding a Notifications migration guide - mostly moving from the Alerts Guide, but also added more context and refs to other information. 
And fixed a bit of formatting.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## How Has This Been Tested?

Reviewed RTD on this PR.

### CODE REVIEW

**Review by abowersox-cb — APPROVED** [review]

File: docs/notifications-migration.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Notifications to Alerts Migration
+=================================
+
+Use this guide to update from using ```get_notifications()```, which leverages the
+```/integrationServices/v3/notification``` API to using Alerts in SDK v1.5.0 or higher with Alerts v7 API.
+
+.. note::
+    The /integrationServices/v3/notificationf API is deprecated, and deactivation is planned for 31 October 2024.
+
+    The Access Level Type SIEM used to access the Notifications API is also deprecated, and deactivation is planned for 31 January 2025.
```
Comment (avanbrunt-cb) [inline docs/notifications-migration.rst:12]: I'm confused with the 31 January 2025 date as 31 October 2024 seems to be the actual date

File: docs/notifications-migration.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
+.. _notifications-migration-guide:
+
+Notifications to Alerts Migration
+=================================
+
+Use this guide to update from using ```get_notifications()```, which leverages the
+```/integrationServices/v3/notification``` API to using Alerts in SDK v1.5.0 or higher with Alerts v7 API.
+
+.. note::
+    The /integrationServices/v3/notificationf API is deprecated, and deactivation is planned for 31 October 2024.
```
Comment (avanbrunt-cb) [inline docs/notifications-migration.rst:10]: typo `notificationf`

File: docs/notifications-migration.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    >>> alerts = api.select(Alert).set_minimum_severity(7).\
+    >>>     add_criteria("type", ["CB_ANALYTICS", "DEVICE_CONTROL"]).\
+    >>>     add_criteria("device_policy", "Standard")
+
+An Alert contains a lot more information than a Notification, and most of the fields are available for searching.
+
+The other modification required is that where the Notifications was a read one queue, Alerts are retrieved using a search.
+An example script with the polling logic implemented is in the GitHub Repository, `alerts_bulk_export.py in GitHub Examples <https://github.com/carbonblack/carbon-black-cloud-sdk-python/tree/develop/examples/platform>`_.
+
+There is also an guide to `Alert Bulk Export <http://localhost:1313/reference/carbon-black-cloud/guides/alert-bulk-export/>`_
```
Comment (avanbrunt-cb) [inline docs/notifications-migration.rst:72]: typo `an guide`

**Review by jclark-vmware — APPROVED** [review]

File: docs/notifications-migration.rst (hunk 1 above)
Comment (kebringer-cb) [reply docs/notifications-migration.rst:12]: Clarified wording.

**Pull request merged**